dev-libs/xalan-c/xalan-c-1.12.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake
```

Comment:
You could create  `sec-keys/openpgp-keys-apache-xalan-c` and then
inherit cmake verify-sig